examples/hello-world-python/helloworld.cpp

```diff
@@ -0,0 +1,30 @@
+// AbiSpack.cpp
```

Comment:
Change to ``helloworld.cpp``.  There is some copyright boilerplate that I've been adding to every file.

Reply:
All set, along with adding the default header.